StringCalculator.java

```diff
@@ -0,0 +1,47 @@
+public class StringCalculator {
+
+    public static void main(String ... args) {
+        StringCalculator stringCalculator = new StringCalculator();
+        System.out.println(stringCalculator.result("1 + 2"));
+    }
+
+
+    public int result(String input) {
+        String[] inputs = input.split(" ");
+        return calculator(inputs.length -1, inputs);
+    }
+
+    public int calculator(int number, String[] input) {
+        if (number == 0) {
+            return Integer.parseInt(input[number]);
+        }
+
+        return branchMethod(number, input);
+    }
+
+    private int branchMethod(int number, String[] input) {
+        switch (input[number]) {
+            case "+" : return add(number, input);
+            case "-" : return subtract(number, input);
+            case "*" : return multiply(number, input);
+            case "/" : return divide(number, input);
+            default: return number;
+        }
+    }
+
+    private int add(int number, String[] input) {
+        return calculator(number -1, input) + Integer.parseInt(input[number + 1]);
```

Comment:
```suggestion
        return calculator(number - 1, input) + Integer.parseInt(input[number + 1]);
```

이 부분을 포함해서, -1 이 들어간 부분에 모두 띄어쓰기가 하나 빠진 것 같아요!